### ISSUE TITLE
TSA.gov_9_2015 - Scen 2 (Spray Paint) - "Search" doesn't Search and "?" opens another window in my browser

### ISSUE THREAD

**Issue by kcropper** [GSA/Crowdsource-Testing-TSA.GOV_9-2015#8]
Issue Name: TSA.gov_9_2015 - Scen 2 (Spray Paint) - "Search" doesn't Search and "?" opens another window in my browser
Device Make: Apple
Device Model: iPhone 5
Device Operating System Version: iOS 8.3
Browser: Safari
Carrier (or WiFi): WiFi
Severity: High
Description:
"Search" button on keyboard doesn't cause it to search; > button on keyboard takes me to another entry field on screen. Not until I zoom out and scroll over to the "?" button on the screen does it then <em>open another window!!</em> for apps.tsa.dhs.gov and show the "Search Results For: spray paint" - and at the top of the window, it shows me the ad suggesting I use the TSA App.  To further complicate matters, when I touch in the field to "enter item or keywords", it zooms in (why?) so much that I can only see "When I fly can I brin" - which means the "?" button is no longer visible.

**Comment by kcropper** [comment]
Another issue: the search is literal - "spary paint" (misspelled "spray") returns "Item Not Found" (in the other window) with a big red X that looks exactly like the red circle with slash of "Not Permitted" (is that as a "just in case"?) and suggests I check the spelling, rather than suggesting possible matches.  When I entered "paint" (by itself) it at least asked if I meant "aerosol paint" or "paint".  

Then when I selected the (small for a touch screen) radio button for one, the next step buttons were "Choose Item" (I just did!) or "Search Again" (Oh, run another search using the item I just selected?)

**Comment by hollyallen** [comment]
I encountered these problems also and agree that this behavior is terrible.

**Comment by hollyallen** [comment]
This image demonstrates the behavior "when I touch in the search field to "enter item or keywords", it zooms in so much that I can only see "When I fly can I bring m" - which means the "?" button is no longer visible."
![img_0005](https://cloud.githubusercontent.com/assets/3476681/10107328/278db428-636f-11e5-855a-a52e53965944.jpg)
